src/helpers/joiResponse.js

```diff
@@ -1,13 +1,22 @@
 import Response from './sendResponse';
 import code from './statusCode';
 
-const joiResponse = (reqToValidate, res, schema, next) => {
-  const { error } = schema.validate(reqToValidate);
+/**
+* @description this method validate user result
+* @param {object} req
+* @param {object} res
+* @param {object} schema
+* @param {object} next
+* @returns {object} error
+* @memberof JoiValidator
+*/
+const joiResponse = (req, res, schema, next) => {
+  const { error } = schema.validate(req.body);
   if (error) {
     return Response.error(
       res,
       code.badRequest,
-      error.details[0].message.replace(/\"/g, '')
+      error.details[0].message.replace(/"/g, '')
```

Comment:
I think you need to update this to [error.details[0].message.replace(/\"/g, '')] in order to send a clean joi response.